python/beeai_framework/backend/chat.py

```diff
@@ -215,25 +213,40 @@ async def executor(_: RetryableContext) -> ChatModelStructureOutput:
             )
         ).get()
 
-    def create(self, chat_model_input: ModelLike[ChatModelInput]) -> Run[ChatModelOutput]:
-        input = to_model(ChatModelInput, chat_model_input)
+    def create(
+        self,
+        messages: list[Message],
+        tools: list[Message] | None = None,
```

Comment:
Use `*` to force keyword arguments to preserve backward compatibility if we change the order in the future